docs/examples/basic_meter/basic_metrics.py

```diff
@@ -42,24 +42,24 @@
 
 # The Meter is responsible for creating and recording metrics. Each meter has a
 # unique name, which we set as the module's name here.
-meter = metrics.get_meter(__name__)
+accumulator = metrics.get_meter(__name__)
```

Comment:
Getting an accumulator from `get_meter` seems so weird. As a user, I wouldn't want to be bombarded by more terms. I'm wondering why we the specs needs to introduce a new word to represent the instrumentation. We don't have this for `Tracer`.

Reply:
> As a user, I wouldn't want to be bombarded by more terms.

yup, you do raise a good point. But IMO if we're renaming the Meter class to Accumulator to comply with the spec, I think we also need to update its usages as well. Otherwise it'll be even more confusing because `meter` is an interface and `meter = metrics.get_meter()`  gives the impression that you're directly instantiating an object of the interface. 

> Getting an accumulator from `get_meter` seems so weird

lol you're right, I wonder if we can fix this incongruity between method name and return type :thinking:

This _kind_ of makes sense to me because `meter` is the super type of `accumulator` so you're still getting a meter from the method. That said, I can also see how `get_meter()` makes it sound like you're explicity expecting a `meter` object. 

The Go repo actually renamed `get_meter()`  to `meter()` [CHANGELOG](https://github.com/open-telemetry/opentelemetry-go/blob/master/CHANGELOG.md#changed-16). Do you think it would help add clarity to make the same change here? This way, with `metrics.meter()`,  you're just accessing whichever meter-type object the `meterprovider` is exposing (which could be anything since meter is just an interface)

Do you have another approach in mind?

Reply:
I don't believe we should be following the Go implementation as the source of truth, rather the [api specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#obtaining-a-meter) say `GetMeter`. I don't know if this is outdated but we can change it if it changes in the future.

I think this whole naming thing is weird in the specs itself, but it's fine right now to adhere to it. Just don't be surprised if there are changes in the future :D

Reply:
Alright makes sense, i'll revert the usage change

Reply:
My understanding is that Accumulator doesn't have to be the SDK meter, but that is an implementation detail of the Python and Go SDKs (see this comment thread https://github.com/open-telemetry/opentelemetry-specification/pull/880#discussion_r482245369)

I think it's even still a bit confusing if `type(metrics.get_meter())` is `Accumulator` instead of `Meter`

Reply:
A best of both worlds approach (and this may require spec changes) may be to add a new Meter object, whose only job is to create instruments that bind to the accumulator.

IMO this is actually a fairly clean break: Meter owns the logic required create new instruments, and accumulation can be the first part of the processing pipeline.